grace/training/train.py

```diff
@@ -12,6 +13,12 @@
 
 from torch.utils.tensorboard import SummaryWriter
 
+logging.basicConfig(
```

Comment:
I think you only have to do this config once, in the run and it is saved for everywhere else the logger is called. In affinityvae we do it in the [run.py script](https://github.com/alan-turing-institute/affinity-vae/blob/develop/run.py)

Reply:
Addressed now, logger config specifies in the `run.py` exclusively 😃